processfamily/win32Popen.py

```diff
@@ -1,3 +1,16 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+native_int = int
+
+from future import standard_library
+standard_library.install_aliases()
```

Comment:
@rorygrandin install_aliases is not appropriate for a library to run

Reply:
modernise is a much safer transform that doesn't have unexpected side effects

https://python-modernize.readthedocs.io/en/latest/

Reply:
I think @graingert is quite right here - especially on the install_aliases side of things...

Reply:
I logged an issue: https://github.com/j5int/processfamily/issues/24